plugins/Overlay/Controller.php

```diff
@@ -235,4 +251,10 @@ protected function outputCORSHeaders()
         $corsHandler = new CORSHandler();
         $corsHandler->handle();
     }
+
+    private function apiRequest($methodName)
+    {
+        $_GET['method'] = $_POST['method'] = 'Overlay.' . $methodName;
```

Comment:
is it possible to use `Request::processRequest` instead of dispatching new request fully?

Reply:
I could use `new Request(...)` (processRequest hard codes some params)..